beacon-chain/db/validator.go

```diff
@@ -72,9 +97,32 @@ func (db *BeaconDB) DeleteValidatorIndex(pubKey []byte) error {
 	})
 }
 
+// HasValidators checks if a validator index map exists out of a list of public keys.
+func (db *BeaconDB) HasValidators(pubKeys [][]byte) (bool, [][]byte) {
```

Comment:
Why does this return a `[][]byte`?

Reply:
It was a suggestion in one of the review comments to return true or false and a list of validators public keys that haven't been found

Reply:
It looks like you never use it and it doesn’t fit with the Boolean name “HasValidators”. Please remove

Reply:
removed